easyrsa3/easyrsa

```diff
@@ -674,6 +680,79 @@ current CA keypair. If you intended to start a new CA, run init-pki first."
             fi
         fi
 	fi
+
+	# Generate the CA with AES256
```

Comment:
what is this comment for? Doesn't make sense.

Reply:
I will correct the indentation in a final (hopefully) version of this patch.

Reply:
It is an artifact from https://github.com/OpenVPN/easy-rsa/blob/243bb708b02897c563631ce62633d90ae80c82ff/easyrsa3/easyrsa#L665

What would you suggest ?

Reply:
I believe I can correct this comment. How does this look:
`	# Choose SSL Library version (1 or 3) and build CA
`

Reply:
I'm submitting a v3, hopefully this comments is suitable.